server/controllers/index.js

```diff
@@ -109,8 +109,10 @@ export const explore = async(ctx, next) => {
   const articleStubs = await getArticleStubs(50);
   const grouped = articleStubs.data.groupBy(stub => stub.headline.indexOf('A drop in the ocean:') === 0);
   const theRest = grouped.first();
-  const topPromo = PromoFactory.fromArticleStub(theRest.first(), 'lead');
-  const second3Promos = theRest.slice(1, 4).map(PromoFactory.fromArticleStub);
+  const topPromo = Object.assign({}, PromoFactory.fromArticleStub(theRest.first(), 'lead'), {modifiers: ['on-cream']});
```

Comment:
It didn't complain here (but broke in the article model where I'd not updated the `type` property to `contentType`. Have updated now.

Actually, this file has `// TODO: FlowType this module` at the top